### ISSUE TITLE
`start --goal=rebase-with-history` put wrong branch on top

### ISSUE THREAD

**Issue by jyasskin** [mhagger/git-imerge#67]
I ran `git checkout service-worker-apps && git imerge start --first-parent --goal=rebase-with-history --name service-worker-apps lkgr^` and got the result at https://github.com/jyasskin/chromium/tree/service-worker-apps.broken.imerge. This was quite possibly my mistake, but I think it's inconsistent with the result I would have gotten from `git rebase lkgr^`. Specifically, `imerge` appears to have put the `lkgr` branch on top of the `service-worker-apps` branch, while a `rebase` would have put `service-worker-apps` on top of `lkgr`.

**Comment by mhagger** [comment]
I agree that this is confusing. It is a historical artifact from the way `git imerge start` was originally implemented. I think the best way to solve this problem would be to add a command `git imerge rebase-with-history` analogous to `git imerge rebase` and `git imerge merge`.
